modules/custom/az_dam/az_dam.info.yml

```diff
@@ -0,0 +1,6 @@
+name: 'Quickstart DAM'
+type: module
+core_version_requirement: ^9
+package: The University of Arizona
+dependencies:
+  - intelligencebank:intelligencebank
```

Comment:
```suggestion
  - ib_dam:ib_dam
  - ib_dam:ib_dam_media
  - ib_dam:ib_dam_wysiwyg

```

Reply:
Shouldn't these be in this format?
```
  - intelligencebank:ib_dam
  - intelligencebank:ib_dam_media
  - intelligencebank:ib_dam_wysiwyg
```

Reply:
@trackleft Shouldn't these be in this format?
```
  - intelligencebank:ib_dam
  - intelligencebank:ib_dam_media
  - intelligencebank:ib_dam_wysiwyg
```

Reply:
It worked the other way, but I'm really not sure, the composer.json file calls it drupal/ib_dam
https://git.drupalcode.org/project/intelligencebank/-/blob/8.x-2.x/composer.json#L2